_tests_/setup.sh

```diff
@@ -0,0 +1,10 @@
+
+echo "> Cloning FHIR Generated Patients..."
+git clone --single-branch --branch master https://github.com/projecttacoma/fhir-patient-generator.git
+
+echo "> Cloning connectathon repo"
+git clone --single-branch --branch master https://github.com/projecttacoma/connectathon.git  
+
+echo "> running the tests"
+node ./run_tests.ts
```

Comment:
Running the tests should happen in a follow on PR.

Reply:
Remove this. since the file does not exist. Just the cloning of the repos is needed. Anyways it would be good to separate the running of the tests from the setup work.